test/fixture/source-map-pragma-exception.js

```diff
@@ -0,0 +1,6 @@
+const test = require('../../');
```

Comment:
I don't think you meant to include this file. It was already removed in https://github.com/sindresorhus/ava/commit/386416846bdb7b8aae84e1b0d8cf77da0bb31fa4.